mcy/src/components/Layout/Drawer.jsx

```diff
@@ -1,5 +1,127 @@
+import * as React from "react";
+import { useNavigate } from "react-router-dom";
+import { Stack, styled } from "@mui/material";
+import Box from "@mui/material/Box";
+import DrawerSlide from "@mui/material/Drawer";
+import Button from "@mui/material/Button";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+import MenuIcon from "@mui/icons-material/Menu";
+import ClearIcon from "@mui/icons-material/Clear";
+
+import DrawerCheck from "../../image/drawer_check.png";
+
 const Drawer = () => {
-  return <div>Drawer</div>;
+  const [open, setOpen] = React.useState(false);
+  const Navigate = useNavigate();
+
+  const MenuList = ["소식", "출석", "생일"];
+
+  const toggleDrawer = (newOpen) => () => {
+    setOpen(newOpen);
+  };
+
+  const pageMove = (index) => {
+    // switch (index) {
+    //   case 0:
+    //     Navigate("/news");
+    //     break;
+    //   case 1:
+    //     Navigate("/attendance");
+    //     break;
+    //   case 2:
+    //     Navigate("/birthDay");
+    //     break;
+    // }
+    index % 3 === 0
+      ? Navigate("/news")
+      : index % 2 === 1
+      ? Navigate("/attendance")
+      : Navigate("/birthDay");
+  };
+
+  const DrawerList = (
+    <DrawerWapper rol="presentation">
+      <DrawerTop>
+        <ExitIcon onClick={toggleDrawer(false)} />
+      </DrawerTop>
+      <List>
+        {MenuList.map((text, index) => (
+          <ListItem key={text} disablePadding>
+            <ListButton onClick={() => pageMove(index)}>
+              <ListIcon>
+                <img src={DrawerCheck} style={{ width: "20px" }} />
+              </ListIcon>
+              <ListText primary={text} />
+            </ListButton>
+          </ListItem>
+        ))}
```

Comment:
```suggestion
            <ListItem onClick={handleGoToNews}>소식</ListItem>
            <ListItem onClick={handleGoToAttendance}>출석</ListItem>
            <ListItem onClick={() => {Navigate("/birthDay")} }>생일</ListItem>
```
여기는 복잡하게 할 필요없이 그냥 이런식으로 이동할수있는 페이지 목록을 표시해주면됨.